src/themes/hawaiian/basic/color/text-icon.json

```diff
@@ -1,7 +1,7 @@
 {
   "basic": {
     "color": {
-      "textIcon": {
+      "texticon": {
```

Comment:
Im seeing both `texticon` and `text-icon` tokens in the hawaiian CSSProperties file